app/controllers/uploads_controller.rb

```diff
@@ -9,42 +9,42 @@ def new
     @upload = Upload.from_csv_type(params[:csv_type])
 
     if @upload.csv_type_check?
-      @requirements = requirements_messages
+      csv_requirements
+
       return
```

Comment:
Not related to PR necessarily, but I'm trying to better understand.

What does this route return for an http response? Also, what does a blank `return` do from within a controller?

Reply:
I updated it so it makes more sense with the if statement returning the csv_requirements